ompy/vector.py

```diff
@@ -173,14 +175,22 @@ def save(self, path: Union[str, Path],
         if filetype is None:
             filetype = filetype_from_suffix(path)
         filetype = filetype.lower()
+
         if filetype == 'numpy':
-            save_numpy_1D(vector.values, vector.E, path)
+            save_numpy_1D(vector.values, vector.E, vector.std, path)
         elif filetype == 'txt':
-            save_txt_1D(vector.values, vector.E, path, **kwargs)
+            save_txt_1D(vector.values, vector.E, vector.std, path, **kwargs)
         elif filetype == 'tar':
-            save_tar([vector.values, vector.E], path)
+            if vector.std is not None:
+                save_tar([vector.values, vector.E, vector.std], path)
+            else:
+                save_tar([vector.values, vector.E], path)
         elif filetype == 'mama':
             mama_write(self, path)
+            if self.std is not None:
+                warnings.warn("MaMa cannot store std. Consider using another format")
```

Comment:
suggestion: "[...] using another format or saving the std. as a separate file"

Reply:
Fixed